TECH-OVERSIGHT-COMMITTEE.md

```diff
@@ -3,41 +3,99 @@
+5. The Steering Committee, with assistance from the current TOC, shall evaluate candidates against the following criteria, assessed over the preceding 12 months. A candidate must satisfy the criteria for "Chop Wood, Carry Water" and at least one additional criterion.
+
+  <ul style="list-style-type:none;">
+      <dl>
+          <dt><b><i>Chop Wood, Carry Water:</i></b></dt>
+          <dd>Demonstrates visibility in technical and collaborative aspects of the project, including but not limited to
+              weekly working group meetings, monthly TOC meetings, and Slack discussions.</dd>
+          <dt><b><i>Senior Maintainer:</i></b></dt>
+          <dd>Has authored and merged at least 20 substantial pull requests (PRs).</dd>
+          <dt><b><i>User Champion:</i></b></dt>
+          <dd>
+              Has resolved at least 5 user-reported issues with pull requests (PRs). Demonstrates active engagement in Slack channels or GitHub issues/discussions in support of users.
+          </dd>
+          <dt><b><i>Project Manager:</i></b></dt>
+          <dd>
+              Demonstrates active involvement in organizing the project’s daily operations, including bug triage, roadmap
+              qualification and definition, leading meetings, driving consensus, and organizing community events.
+          </dd>
+          <dt><b><i>Architect:</i></b></dt>
+          <dd>
+              Has produced at least 3 design documents for features incorporated into the project roadmap. Has reviewed at least 10 design documents authored by other community members.
+          </dd>
+      </dl>
+  </ul>
+
+6. Candidates must submit a written application, including a self-assessment against the qualification criteria. The application shall be forwarded to the TOC for confidential feedback and subsequently returned to the Steering Committee.
+7. The Steering Committee shall individually vote on each candidate to determine their suitability for serving on the TOC. An affirmative vote of at least 60% of the seats is required for a candidate to be approved.
+    a. If the number of candidates is fewer than or equal to the number of open seats, this vote shall serve as the election.
+    b. If the number of candidates exceeds the number of open seats, this vote shall qualify candidates to proceed to a second round of voting.
+8. Should there be more qualified candidates than open TOC seats, the Steering Committee shall select the winning candidates via a Condorcet election.
+
+### Vacancies
+
+1. In the event that a TOC member vacates their seat during their term, a by-election shall be conducted to fill the position for the remainder of the term in accordance with the regular qualification and election procedures.
+2. The Steering Committee may initiate a vote of no confidence in a TOC member. The member shall be removed if the motion is approved by at least 60% of the Steering Committee seats. The vacated seat shall be filled through the regular vacancy process.
```

Comment:
Should we make it clear that removal is for a particular seat, so when the seat is filled its for the remainder of that seat's term?  (It's sort of implied, but I think stating "The vacated seat shall be filled for the remainder of its term through the regular vacancy process." would make strengthen it.)

Reply:
SGTM